django/utils/formats.py

```diff
@@ -217,7 +217,7 @@ def localize(value, use_l10n=None):
     elif isinstance(value, datetime.date):
         return date_format(value, use_l10n=use_l10n)
     elif isinstance(value, datetime.time):
-        return time_format(value, "TIME_FORMAT", use_l10n=use_l10n)
+        return time_format(value, use_l10n=use_l10n)
```

Comment:
We could to the same for `date_format` in tests:
https://github.com/django/django/blob/5cf9ff970e08b10d3f7d62734dbce9e8eaeff453/tests/i18n/tests.py#L964

Reply:
👍🏻 Good shout. Added a commit.